pkg/systemd/services/services.jsx

```diff
@@ -53,18 +54,20 @@ const _ = cockpit.gettext;
 superuser.reload_page_on_change();
 
 export const systemd_client = {
-    system: cockpit.dbus("org.freedesktop.systemd1", { bus: "system", superuser: "try" }),
-    user: cockpit.dbus("org.freedesktop.systemd1", { bus: "session" }),
+    system: cockpit.dbus(s_bus.BUS_NAME, { bus: "system", superuser: "try" }),
+    user: cockpit.dbus(s_bus.BUS_NAME, { bus: "session" }),
 };
 const timedate_client = cockpit.dbus('org.freedesktop.timedate1');
 export let clock_realtime_now;
 export let clock_monotonic_now;
 
-export const SD_MANAGER = "org.freedesktop.systemd1.Manager";
-export const SD_OBJ = "/org/freedesktop/systemd1";
-
 export const MAX_UINT64 = 2 ** 64 - 1;
 
+function debug() {
+    if (window.debugging == "all" || window.debugging?.includes("services"))
+        console.debug.apply(console, arguments);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230303-103127-ffd636d2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)